lib/Target/SubstraitPB/Import.cpp

```diff
@@ -32,6 +33,30 @@ namespace pb = google::protobuf;
 
 namespace {
 
+// Copied from
+// https://github.com/llvm/llvm-project/blob/dea33c/mlir/lib/Transforms/CSE.cpp.
```

Comment:
Could we just reuse it?

Reply:
Unfortunately, no, at least not currently. It's in the `cpp` file, so we'd have to move it upstream to some header to make it accessible.